cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaReconciler.java

```diff
@@ -1163,6 +1175,118 @@ protected Future<Void> clusterId(KafkaStatus kafkaStatus) {
+
+        return ReconcilerUtils.clientSecrets(reconciliation, secretOperator)
+                .compose(compositeFuture -> {
+                    Promise<Void> resultPromise = Promise.promise();
+                    vertx.createSharedWorkerExecutor("kubernetes-ops-pool").executeBlocking(
+                            future -> {
+                                Admin kafkaAdmin = null;
+                                final Future<Collection<TopicDescription>> descriptions;
+                                try {
+                                    String bootstrapHostname = KafkaResources.bootstrapServiceName(reconciliation.name()) + "." + reconciliation.namespace() + ".svc:" + KafkaCluster.REPLICATION_PORT;
+                                    LOGGER.debugCr(reconciliation, "Creating AdminClient for clusterId using {}", bootstrapHostname);
+                                    kafkaAdmin = adminClientProvider.createAdminClient(bootstrapHostname, compositeFuture.resultAt(0), compositeFuture.resultAt(1), "cluster-operator");
+
+                                    Admin finalKafkaAdmin = kafkaAdmin;
+                                    Future<Set<String>> topicNames = topicNames(finalKafkaAdmin);
+
+                                    descriptions = topicNames.compose(names -> {
+                                        LOGGER.debugCr(reconciliation, "Got {} topic names", names.size());
+                                        LOGGER.traceCr(reconciliation, "Topic names {}", names);
+                                        return describeTopics(finalKafkaAdmin, names);
+                                    });
+
+                                    Future<Boolean> topicsOnGivenBroker = descriptions
+                                            .compose(topicDescriptions -> {
+                                                LOGGER.debugCr(reconciliation, "Got {} topic descriptions", topicDescriptions.size());
+                                                return Future.succeededFuture(brokerHasAnyReplicas(topicDescriptions, podId, kafkaStatus));
+                                            }).recover(error -> {
+                                                LOGGER.warnCr(reconciliation, "failed to get topic descriptions", error);
+                                                return Future.failedFuture(error);
+                                            });
+
+                                } catch (KafkaException e) {
+                                    LOGGER.warnCr(reconciliation, "Kafka exception getting clusterId {}", e.getMessage());
+                                } finally {
+                                    if (kafkaAdmin != null) {
+                                    }
+                                }
+                                future.complete();
+                            }, true, resultPromise);
+                    return resultPromise.future();
+                });
+    }
+
+
```

Comment:
You have in one place no empty lines, in other place two empty lines. Maybe we can use one empty line everywhere ;-)